_clubs/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Clubs and Groups
+date: 2019-06-10 07:55
+author: Klahanie
+comments: true
+categories: [clubs]
+show-title: true
+---

```

Comment:
Hope I'm in the right place for this edit: "Klahanie's natural areas ." (missing a period) I'm also not a fan of the "!" at the end of the Klahanie Sharks sentence; a period is sufficient.

Reply:
While all very cool in their own right, the individual pages found under Clubs and Groups do not have a consistent look and feel. Is this okay?
Also, is there any chance that residents will want to navigation between the different clubs and groups? If so, it might be nice to add that ability, so folks don't have to return to the left nav bar each time they want to see a different group/club.

Reply:
I think the differences in what's on each club page are ok. They're all following the same template, but the content is basically left to those groups.